vip-jetpack/connection-pilot/class-jetpack-connection-controls.php

```diff
@@ -168,16 +168,12 @@ public static function connect_akismet(): bool {
 	 * @return bool|\WP_Error True if site is connected, error otherwise.
 	 */
 	public static function connect_vaultpress() {
-		if ( class_exists( 'VaultPress' ) ) {
-			$vaultpress = \VaultPress::init();
-			if ( ! $vaultpress->is_registered() ) {
-				return $vaultpress->register_via_jetpack( true );
-			}
-
-			return true;
```

Comment:
The only acceptable case in which VaultPress is not defined should be when it's skipped, which is already checked in the caller. If not, we'll fatal, but it is fine since we'd be detecting an edge case.